editor/plugins/texture_region_editor_plugin.cpp

```diff
@@ -57,8 +57,11 @@ void TextureRegionEditor::_region_draw() {
 		base_tex = obj_styleBox->get_texture();
 	else if (atlas_tex.is_valid())
 		base_tex = atlas_tex->get_atlas();
-	else if (tile_set.is_valid() && selected_tile != -1)
-		base_tex = tile_set->tile_get_texture(selected_tile);
+	else if (tile_set.is_valid() && selected_tile != -1) {
+		if (tile_set->has_tile(selected_tile))
```

Comment:
I guess you could put this additional condition on the main one after `&&` ?

Reply:
Fixed :)